.devcontainer/scripts/start.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Performs setup tasks _after_ the .devcontainer Docker container is created.
+
+# Install Node dependencies.
+yarn install
```

Comment:
would we need to install `yarn` in the container first? not clear if the base image comes with it @disposedtrolley

Reply:
It comes with it! I think MS graciously installs it as part of the NodeJS base image for dev containers haha